fabtests/unit/mr_cache_evict.c

```diff
@@ -591,8 +721,15 @@ static void usage(void)
 		"allocation is returned. This can be used to verify the \n"
 		"underlying physical memory changes between MMAP, BRK, and \n"
 		"SBRK allocations. When running as non-root, the reported \n"
-		"physical address is always zero.");
+		"physical address is always zero.\n\n"
+		"This test can also be used to verify caching of device memory\n"
+		"regions. Currently, FI_HMEM_CUDA is supported. The env variable\n"
+		"FI_MR_CUDA_CACHE_MONITOR_ENABLED can be used to disable the\n"
```

Comment:
```
Setting the env variable FI_MR_CUDA_CACHE_MONITOR_ENABLED to false will disable...
```
Though it would be best to avoid directly referencing the environment variable by name.  If the name changes (likely as we define a standard way to enable/disable all monitors), this print statement buried in fabtests is almost guaranteed never to be updated.